src/ResourceStore.js

```diff
@@ -67,7 +67,7 @@ class ResourceStore extends EventEmitter {
     for (const m in resources) {
       if (
         typeof resources[m] === 'string' ||
-        Object.prototype.toString.apply(resources[m]) === '[object Array]'
+        /[object (Object|Array)]/.test(Object.prototype.toString.apply(resources[m]))
```

Comment:
Main change is here. Not sure why the above file was also modified when I ran `npm run build`